extensions/applicationinsights-debugplugin-js/sample/public/sample.js

```diff
@@ -0,0 +1,124 @@
+if (!Object.keys) {
+  Object.keys = (function() {
+    'use strict';
+    var hasOwnProperty = Object.prototype.hasOwnProperty,
+        hasDontEnumBug = !({ toString: null }).propertyIsEnumerable('toString'),
+        dontEnums = [
+          'toString',
+          'toLocaleString',
+          'valueOf',
+          'hasOwnProperty',
+          'isPrototypeOf',
+          'propertyIsEnumerable',
+          'constructor'
+        ],
+        dontEnumsLength = dontEnums.length;
+
+    return function(obj) {
+      if (typeof obj !== 'function' && (typeof obj !== 'object' || obj === null)) {
+        throw new TypeError('Object.keys called on non-object');
+      }
+
+      var result = [], prop, i;
+
+      for (prop in obj) {
+        if (hasOwnProperty.call(obj, prop)) {
+          result.push(prop);
+        }
+      }
+
+      if (hasDontEnumBug) {
+        for (i = 0; i < dontEnumsLength; i++) {
+          if (hasOwnProperty.call(obj, dontEnums[i])) {
+            result.push(dontEnums[i]);
+          }
+        }
+      }
+      return result;
+    };
+  }());
+}
+
+function myFunc() {
+
+  var toTrack = [
+    'trackEvent',
+    'trackPageView',
+    'trackPageViewPerformance',
+    'trackException',
+    'trackTrace',
+    'trackMetric',
+    'trackDependencyData',
+    'throwInternal',        // called when a message is logged internally
+    'logInternalMessage',   // called when a message is logged internally
+    'triggerSend',          // called when data is queued to be sent to the server
+    '_sender',              // called when data is sent to the server
+  ];
+
+  var debugPlugin = Microsoft.ApplicationInsights.DebugPlugin.DebugPlugin;
```

Comment:
Please add a note in the readme that this is a BETA release and that there may be future breaking changes such as name will most likely change.